src/main/java/duke/Storage.java

```diff
@@ -1,3 +1,5 @@
+package duke;
+
 import java.io.*;
```

Comment:
Perhaps could try to minimize wildcard imports? To fix this fast, delete this statement, and right click all the red squiggly lines and let intelli-j auto import for you.
IntelliJ might automatically wildcard import if you have more than 3 classes, so need to uncheck that option